packages/react/src/components/ExpandCollapsePanel/ExpandCollapsePanel.test.tsx

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { default as ExpandCollapsePanel, PanelTrigger } from './';
+import { SinonStub, createSandbox } from 'sinon';
+import * as stylesheets from '../../utils/stylesheets';
+
+const sandbox = createSandbox();
+const noop = () => {
+  // not empty
+};
+
+let matchMediaStub: SinonStub;
+
+beforeEach(() => {
+  window.matchMedia = window.matchMedia || noop;
+  matchMediaStub = sandbox.stub(window, 'matchMedia').returns({
+    matches: false
+  } as MediaQueryList);
+});
+
+afterEach(() => {
+  jest.resetAllMocks();
+  sandbox.restore();
+});
+
+export const isVisible = (element: HTMLElement): boolean => {
+  const node = element.parentElement;
+  if (!node) {
+    throw new Error('Element has no parent');
+  }
+  // Assuming that the 'is--hidden' class is used to hide the element0и
+  return !node.classList.contains('is--hidden');
+};
+
+test('should render children', () => {
+  render(
+    <ExpandCollapsePanel>
+      <div>Hello World</div>
+    </ExpandCollapsePanel>
+  );
+
+  expect(screen.getByText('Hello World')).toBeInTheDocument();
+});
+
+test('should render multiple children', () => {
+  render(
+    <ExpandCollapsePanel>
+      <div>blue</div>
+      <div>green</div>
+    </ExpandCollapsePanel>
+  );
+
+  expect(screen.getByText('blue')).toBeInTheDocument();
+  expect(screen.getByText('green')).toBeInTheDocument();
+});
+
+test('should passthrough props', () => {
+  const props = { foo: 'bar' };
```

Comment:
Similar to one of the other comments I made in a different PR, we should use `ExpandCollapsePanelProps` here because `foo` isn't an acceptable prop.